helm_deploy/use-of-force/values.yaml

```diff
@@ -3,3 +3,6 @@
 
 generic-prometheus-alerts:
   targetApplication: use-of-force
+
+  ingress:
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
```

Comment:
Not sure this is right? 

Don't think it should be nested under `generic-prometheus-alerts`, needs to be inside an annotations section

Reply:
fixed